apps/preview/nuxt/pages/showcases/OrderSummary/OrderSummary.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="md:shadow-lg md:rounded-md md:border md:border-neutral-100">
+    <div class="flex justify-between items-end bg-neutral-100 md:bg-transparent py-2 px-4 md:px-6 md:pt-6 md:pb-4">
+      <p class="typography-headline-4 font-bold">Order summary</p>
+      <p class="typography-text-base font-semibold">(Items: {{ orderDetails.items }})</p>
+    </div>
+    <div class="px-4 pb-4 mt-3 md:px-6 md:pb-6 md:mt-0">
+      <div class="flex justify-between typography-text-base pb-4">
+        <div class="flex flex-col grow pr-2">
+          <p>Items Subtotal</p>
+          <p class="ml-auto typography-text-xs text-neutral-500">Original Price</p>
+          <p class="ml-auto typography-text-xs text-secondary-700">Savings</p>
+          <p class="my-2">Delivery</p>
+          <p>Estimated Sales Tax</p>
+        </div>
+        <div class="flex flex-col text-right">
+          <p>{{ formatPrice(itemsSubtotal) }}</p>
+          <p class="typography-text-xs text-neutral-500">{{ formatPrice(orderDetails.originalPrice) }}</p>
+          <p class="typography-text-xs text-secondary-700">{{ formatPrice(orderDetails.savings) }}</p>
+          <p class="my-2">{{ formatPrice(orderDetails.delivery) }}</p>
+          <p>{{ formatPrice(orderDetails.tax) }}</p>
+        </div>
+      </div>
+      <div v-if="promoCode" class="flex items-center py-4 border-t border-neutral-200">
+        <p>PromoCode</p>
+        <SfButton size="sm" variant="tertiary" class="ml-auto mr-2" @click="removePromoCode">Remove</SfButton>
+        <p>{{ formatPrice(promoCode) }}</p>
+      </div>
+      <div class="flex gap-x-2 py-4 border-y border-neutral-200 mb-4">
+        <SfInput v-model="inputValue" placeholder="Enter promo code" wrapper-class="grow" />
+        <SfButton variant="secondary" @click.prevent="checkPromoCode">Apply</SfButton>
```

Comment:
```suggestion
        <SfButton variant="secondary" @click="checkPromoCode">Apply</SfButton>
```
question: Is there any particular reason why `prevent` modifier was added to `click` event?

Reply:
no, I was checking something and it left :) Removed https://github.com/vuestorefront/storefront-ui/pull/2579/commits/0857b9464bd93c263ac286d9636be4fc976cf218